requirements.txt

```diff
@@ -1,7 +1,8 @@
-google==3.0.0
-googleapis-common-protos==1.53.0
-setuptools==65.5.1
-grpcio==1.53.0
-grpcio-tools==1.47.0
+google>=3.0.0
+googleapis-common-protos>=1.53.0
+setuptools>=65.5.1
+grpcio>=1.53.0
+grpcio-tools>=1.47.0
```

Comment:
or that we support one version or that we add the max version and min version.
for example - dbt-common>=1.13.0,<2.0
you should get the maximum version that you need for the DBT and verify that you don't break the sqlAlchemy and Airflow.